environments/kubernetes/main.jsonnet

```diff
@@ -2,7 +2,16 @@ local app =
   (import 'kube-thanos.libsonnet') +
   (import 'telemeter.libsonnet') +
   (import 'observatorium.libsonnet') +
-  (import 'jaeger.libsonnet');
+  (import 'jaeger.libsonnet') + {
+    thanos+:: {
+      namespace:: $.observatorium.namespace,
+      externalURL:: 'https://observatorium.api.stage.openshift.com',
+
+      querier+:: {
+        externalPrefix: '%s/ui/v1/metrics' % $.observatorium.api.externalURL,
```

Comment:
This _should_ work as a relative path to the root of the domain `/ui/v1/metrics`.